java/src/lib.rs

```diff
@@ -122,8 +97,21 @@ pub extern "system" fn Java_glide_ffi_resolvers_RedisValueResolver_valueFromPoin
     _class: JClass<'local>,
     pointer: jlong,
 ) -> JObject<'local> {
-    let value = unsafe { Box::from_raw(pointer as *mut Value) };
-    redis_value_to_java(&mut env, *value, true)
+    handle_panics(
+        move || {
```

Comment:
Why do we need `move` in `handle_panics`?

Reply:
The JNIEnv needs to be moved into the closure. Otherwise, it'll be borrowed mutably since most methods on the JNIEnv required `&mut`. This then causes compilation errors because `&mut JNIEnv` doesn't implement `UnwindSafe` (as mentioned in this issue https://github.com/jni-rs/jni-rs/issues/432). So if that issue were to be resolved, then we could avoid the `move` here and also simplify the API as a whole.

Reply:
Thanks a lot!